irma/1.0.3/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM --platform=linux/x86_64 ubuntu:focal as app
+
+# ARG sets environment variables during the build stage
+ARG IRMA_VER="1.0.3"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="2"
+LABEL software="IRMA"
+LABEL software.version=$IRMA_VER
+LABEL description="IRMA was designed for the robust assembly, variant calling, and phasing of highly variable RNA viruses. Currently IRMA is deployed with modules for influenza, ebolavirus and coronavirus."
+LABEL website="https://wonder.cdc.gov/amd/flu/irma/"
+LABEL license="https://wonder.cdc.gov/amd/flu/irma/disclaimer.html"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ perl \
+ default-jre \
+ apt-transport-https \
+ ca-certificates \
+ gnupg \
+ libcurl4-openssl-dev \
+ libssl-dev \
+ libxml2 \
+ libxml2-dev \
+ libncurses5 \
+ wget \
+ git && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# IRMA Needs R
+RUN  apt-get update && apt-get -y upgrade && apt-get -y install r-base \
+  r-base-dev
+
+ # install IRMA
+RUN wget https://wonder.cdc.gov/amd/flu/irma/flu-amd-202209.zip && \
+  unzip flu-amd-202209.zip && \
+  rm flu-amd-202209.zip && \
+  chmod 755 -R flu-amd && \
+  mkdir data
+
+
+
+# set the environment
+ENV PATH="${PATH}:/flu-amd"
+
+
+# WORKDIR sets working directory
+WORKDIR /data
+
+
+
+#This software takes fastq files which may to large for the git repository; would like guidance here.
+FROM app as test
+
+RUN cd /flu-amd/tests && \
+  ./test_run.sh
+
+ENV PATH="$PATH:/data/sratoolkit.2.9.2-ubuntu64/bin" \
+    LC_ALL=C
+
+RUN wget --output-document sratoolkit.tar.gz http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/2.9.2/sratoolkit.2.9.2-ubuntu64.tar.gz && \
+  tar -vxzf sratoolkit.tar.gz && \
+  rm sratoolkit.tar.gz && \
+  wget https://sra-pub-run-odp.s3.amazonaws.com/sra/SRR17940172/SRR17940172 && \
+  fastq-dump --split-files SRR17940172
```

Comment:
What if instead of using the SRA toolkit, you just downloaded the files from the (ENA)[https://www.ebi.ac.uk/ena/browser/view/SRR17940172?show=reads]?

```
RUN wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR179/072/SRR17940172/SRR17940172_1.fastq.gz && wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR179/072/SRR17940172/SRR17940172_2.fastq.gz
```

Also, did they fix the fastq file header issue in 1.5? I vaguely remember someone mentioning that they were working on it.

Reply:
Yeah let's get rid of the sra toolkit. But how did you know/find the address? Also I have tested IRMA without altering the headers and it fails.